contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -128,20 +135,27 @@
                 <field name="fast_axes" type="NX_CHAR" recommended="true"/>
                 <field name="slow_axes" recommended="true"/>
                 <group type="NXcollectioncolumn">
+                    <!--
+                        TODO: Comment from Anders Frisk on proposal page
+                        What is the definition of a collection column?
+
+                        Optional constant settings (like lens focusing parameters on the sample: position_x, position_y, working_distance)
+                    -->
                     <field name="scheme">
                         <doc>
                              Scheme of the electron collection column.
                         </doc>
                         <enumeration>
                             <item value="Standard"/>
                             <item value="Angular dispersive"/>
+                            <item value="Spatial dispersive"/>
                             <item value="Selective area"/>
                             <item value="Deflector"/>
                             <item value="PEEM"/>
                             <item value="Momentum Microscope"/>
                         </enumeration>
                     </field>
-                    <field name="mode" recommended="true"/>
+                    <field name="mode" recommended="true"/> <!-- TODO: What does this refer to. It is currently not in the collection column base class -->
```

Comment:
Did we loose this from the base class at some point?

Reply:
It stores the name of the lens mode in use. It used to be part of the base class: https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXcollectioncolumn.html#nxcollectioncolumn

Reply:
This was meant to store the name string of the saved lens setting. Some machines, for example Scienta, do not let normal users access the individual lens voltage values but have fixed lens settings. Specifying this string univocally defines a detector configuration, and it may be the only information a normal user has access to.

Reply:
Is it back in the base class? If so, let's remove the comment and resolve

Reply:
It is in the base class. I remove the doc string from the appdef since it was the same as in the base class.